frontend/src/app/features/admin/types/query-group.component.html

```diff
@@ -1,17 +1,27 @@
 <div class="type-form-conf-group type-form-query-group">
-    <div class="group-head">
-        <span class="group-handle icon-drag-handle"></span>
-        <op-group-edit-in-place
-                [name]="group.name"
-                (onValueChange)="rename($event)"
-                class="group-name">
-        </op-group-edit-in-place>
-        <span class="delete-group icon-small icon-close" (click)="deleteGroup.emit()"></span>
-    </div>
-    <div class="type-form-query">
-        <span class="type-form-query-group--edit-button" (click)="editQuery.emit()">
-          <op-icon icon-classes="button--icon icon-edit"></op-icon>
-            {{ text.edit_query }}
-        </span>
-    </div>
+  <div class="group-head">
+    <span class="spot-icon spot-icon_1_25 spot-icon_drag-handle group-handle"></span>
+    <op-group-edit-in-place
+      [name]="group.name"
+      (onValueChange)="rename($event)"
+      class="group-name"
+    ></op-group-edit-in-place>
+    <button
+      class="delete-group clear-button"
+      type="button"
+      (click)="deleteGroup.emit()"
+    >
+      <span class="spot-icon spot-icon_1 spot-icon_close"></span>
+    </button>
+  </div>
+  <div class="type-form-query">
+    <button
+      class="clear-button type-form-query-group--edit-button"
+      type="button"
+      (click)="editQuery.emit()"
+    >
+      <span class="spot-icon spot-icon_1_25 spot-icon_edit spot-icon_inline"></span>
```

Comment:
so, you use `spot-icon_inline` for buttons, that are not `display: flex`, correct?

Reply:
Specifically if we're in an inline context